python/examples/augmented_diff.py

```diff
@@ -25,9 +25,13 @@
 for block in osc:
     for e in block:
         action_key = e.tag + "/" + e.get("id")
+        # Always ensure we're updating to the latest version of an object for the diff
         if action_key in actions:
-            # check the version TODO make sure this actually works
-            if obj.get("version") > actions[action_key].element.get("version"):
+            newest_version = int(actions[action_key].element.get("version"))
```

Comment:
I think the intention of this check was to verify that when we loop the elements, we always end up with the latest version of an element in our `actions` dict. Is that correct?

Reply:
Yes it looks correct now, I think the way it was before with > was backwards; I don't believe I ever tested this case rigorously.